.github/workflows/helm.yml

```diff
@@ -22,24 +23,33 @@ jobs:
         run: helm lint varnish-operator
       - name: Ensure helm-releases folder exists
         run: mkdir -p helm-releases
+      - name: Get version
+        id: get_version
+        run: |
+          if (( ${{ github.event_name == 'release' }} )); then
+            echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          else
+            echo "VERSION=v0.0.0-${GITHUB_REF##*/}" >> $GITHUB_ENV
```

Comment:
I guess without a version.txt file or something similar, there's no way to know what to put here other than 0.0.0? For example, in icm-cassandra we publish pre-release charts in the form: `cassandra-0.16.5-feature-v4-support-dev.tgz`

Reply:
well, we can fetch the last tag and use that one, but the artifact won't get pushed anywhere. It is needed just to not get the "invalid semver" error. That version won't be visible anywhere outside the build.